packages/create-sitecore-jss/src/templates/nextjs/sitecore.config.ts.example

```diff
@@ -0,0 +1,35 @@
+import { defineConfig } from '@sitecore-content-sdk/nextjs/config';
+/**
+ * @type {import('@sitecore-content-sdk/nextjs/config').SitecoreConfig}
+ * See the documentation for `defineConfig`:
+ * https://doc.sitecore.com/xmc/en/developers/content-sdk/the-sitecore-configuration-file.html
+ */
+export default defineConfig({
+  api: {
+    edge: {
+      contextId:
+        process.env.SITECORE_EDGE_CONTEXT_ID ||
+        process.env.NEXT_PUBLIC_SITECORE_EDGE_CONTEXT_ID ||
+        '',
+      clientContextId: process.env.NEXT_PUBLIC_SITECORE_EDGE_CONTEXT_ID,
+      edgeUrl: process.env.SITECORE_EDGE_URL || process.env.NEXT_PUBLIC_SITECORE_EDGE_URL,
+    },
+    local: {
+      apiKey: process.env.NEXT_PUBLIC_SITECORE_API_KEY || '',
+      apiHost: process.env.NEXT_PUBLIC_SITECORE_API_HOST || '',
+    },
+  },
+  defaultSite: process.env.NEXT_PUBLIC_SITECORE_SITE_NAME,
+  defaultLanguage: process.env.NEXT_PUBLIC_DEFAULT_LANGUAGE || 'en',
+  editingSecret: process.env.JSS_EDITING_SECRET,
+  redirects: {},
+  multisite: {
+    enabled: true,
+    useCookieResolution: () => process.env.VERCEL_ENV === 'preview',
+  },
+  personalize: {
```

Comment:
please add `enabled` to this and `redirects` section.
Overall I think we can go wild with the example and show all the different config options available, unless you agreed differently with Christian.